src/Commands/Extension/AzCommand.cs

```diff
@@ -72,43 +64,28 @@ protected override AzArguments BindArguments(ParseResult parseResult)
 
     private static string? FindAzCliPath()
     {
+        string executableName = "az";
+
         // Return cached path if available and still exists
         if (!string.IsNullOrEmpty(_cachedAzPath) && File.Exists(_cachedAzPath))
         {
             return _cachedAzPath;
         }
 
-        var isWindows = RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
-        var searchPaths = new List<string>(AzureCliPaths);
-
-        // Add PATH environment directories
-        if (Environment.GetEnvironmentVariable("PATH")?.Split(Path.PathSeparator) is { } pathDirs)
-        {
-            searchPaths.AddRange(pathDirs);
-        }
+        var pathEnv = Environment.GetEnvironmentVariable("PATH");
```

Comment:
Is there a reliable xplat 'where' command?

Reply:
I don't think there is a xplat "where" command. And it seems to me this is the c# solution for xplat 'where'. :)

Reply:
You could use `which` on linux but it the proposed refactor is cleaner / more xplat